apache-tomcat-mixin/dashboards/apache-tomcat-overview.libsonnet

```diff
@@ -670,7 +674,7 @@ local logsPanel = {
   grafanaDashboards+:: {
     'apache-tomcat-overview.json':
       dashboard.new(
-        'Apache Tomcat overview',
+        'Apache Tomcat Overview',
```

Comment:
Please revert this change, our style guide says to only capitalise the first word and/or product names :)

Reply:
Will do, sorry about that!